src/pages/Join/JoinInfo.jsx

```diff
@@ -0,0 +1,167 @@
+import React, { useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+import BackButtonContainer from './BackButtonContainer';
+import LoginInput from '../Login/components/LoginInput';
+import Birthday from './Birthday';
+import LoginButton from '../Login/components/LoginButton';
+import './Join.scss';
+
+export default function JoinInfo() {
+  const navigate = useNavigate();
+
+  const [joinUserInfo, setJoinUserInfo] = useState({
+    email: '',
+    password: '',
+    passwordCheck: '',
+    nickname: '',
+    phoneNumber: '',
+    birthday: '2023-01-01',
+  });
+  const joinUserInfoData = {
+    email: joinUserInfo.email,
+    password: joinUserInfo.password,
+    nickname: joinUserInfo.nickname,
+    phoneNumber: joinUserInfo.phoneNumber,
+    birthday: joinUserInfo.birthday,
+  };
+
+  const isVaild =
+    joinUserInfo.email.includes('@') &&
+    joinUserInfo.email.includes('.') &&
+    joinUserInfo.password.length >= 8 &&
+    joinUserInfo.password === joinUserInfo.passwordCheck;
+
+  const handleJoin = () => {
+    if (isVaild) {
+      navigate('/join-done');
+      fetch('data/userData.json', {
+        method: 'POST',
+        headers: {
+          'Content-Type': 'application/json;charset=utf-8',
+        },
+        body: JSON.stringify(joinUserInfoData),
+      }).then();
+    } else if (
+      !joinUserInfo.email.includes('@') ||
+      !joinUserInfo.email.includes('.')
+    ) {
+      alert('이메일 형식을 지켜주세요.');
+    } else if (joinUserInfo.password.length < 8) {
+      alert('비밀번호는 8자리 이상이어야 합니다.');
+    } else if (joinUserInfo.password !== joinUserInfo.passwordCheck) {
+      alert('비밀번호 확인이 일치하지 않습니다!');
+    }
+  };
+
+  const handleBirthdayChange = formattedBirthday => {
+    setJoinUserInfo(prev => {
+      return {
+        ...prev,
+        birthday: formattedBirthday,
+      };
+    });
```

Comment:
개인의 차이일 수 있지만, 저같은 경우는 return문은 앞에 로직이 없으면 사용하지 않습니다. 동철님도 앞으로 개발하시면서 이처럼 동철님 나름의 기준을 세워보세요!
참고만 해주세요!
```suggestion
    setJoinUserInfo(prev => ({
      ...prev,
      birthday: formattedBirthday,
    }));
```

Reply:
저도 평소엔 적지 않는 것을 선호하는 편인데 아무 생각 없이 작성한 거 같습니다...